lua/Application.lua

```diff
@@ -465,6 +471,8 @@ metaTableApplication_ = {
 -- @arg data.loading An optional string with the name of loading icon. The loading available are: "balls",
 -- "box", "default", "ellipsis", "hourglass", "poi", "reload", "ring", "ringAlt", "ripple", "rolling", "spin",
 -- "squares", "triangle", "wheel" (see http://loading.io/). The default value is 'default'.
+-- @arg data.key An optional string with the Google Maps key/authentication.
```

Comment:
"with 39 characters describing the Google Maps key."
Add a link to a webpage on how to get such key.